src/qpm.io/common/package.go

```diff
@@ -112,7 +114,29 @@ func LoadPackage(path string) (*PackageWrapper, error) {
 		}
 		defer file.Close()
 
-		if err = json.Unmarshal(file, pw.Package); err != nil {
+        raw := jsonUtils.RawMessage{}
+        if  err = jsonUtils.NewDecoder(file).Decode(&raw); err != nil {
+            return pw, err
+        }
+        
+        var v interface{}
+        
+        if err = jsonUtils.Unmarshal(raw,&v); err != nil {
+            return pw,err
+        }
+        
+        re := regexp.MustCompile("[-\\.]");
+        
+        var m map[string] interface{};
+        
+        m = v.(map[string]interface{})
+        var license string;
+        license = m["license"].(string);
+        m["license"] = re.ReplaceAllString(license ,"_");
+        
+        raw,err = jsonUtils.Marshal(m);
+
+		if err = json.Unmarshal(bytes.NewReader(raw), pw.Package); err != nil {
```

Comment:
Instead of trying to convert the license strings when unmarshalling the JSON, let's leave them with underscores in the qpm.json file and instead move the conversion to the init.go code where we can add support for people entering a GitHub style string.